crates/hyperqueue/src/common/cli.rs

```diff
@@ -191,7 +191,7 @@ pub enum SubCommand {
     Job(JobOpts),
     /// Commands for tasks
     Task(TaskOpts),
-    /// Submits a new job
+    /// Submit a new job
```

Comment:
We have all cli doc verbs in "s" form.

Reply:
[Not all](https://github.com/it4innovations/hyperqueue/blob/146463615121481aa6d5eab5a26c4899e454a989/crates/hyperqueue/src/common/cli.rs#L376) :laughing: But it looks like I updated it in the wrong "direction", thanks.